src/ripple/app/tests/Offer.test.cpp

```diff
@@ -41,6 +42,15 @@ class Offer_test : public beast::unit_test::suite
         return env.current()->info().parentCloseTime.time_since_epoch().count();
     }
 
+    static auto
+    xrpMinusFee (jtx::Env const& env, std::int64_t xrpAmount)
+        -> jtx::PrettyAmount
+    {
+        using namespace jtx;
+        auto feeDrops = env.current ()->fees ().base;
+        return drops (dropsPerXRP<std::int64_t>::value * xrpAmount - feeDrops);
```

Comment:
Don't confused `jtx::XRP_t` with those other types, the `jtx` class is designed for specifically for tests and making it easy to write certain statements.